### PR TITLE
cornell: add support for Cornell University

### PR DESCRIPTION
**Pull request by willbarkoff**
Add support for Cornell University. See also MyHomeworkSpace/api-server#7

### CODE REVIEW

File: app/schools/EnrollModal.jsx [hunk 1]
```diff
@@ -23,7 +24,8 @@ import Modal from "ui/Modal.jsx";
 
 var enrollComponents = {
 	dalton: DaltonEnroll,
-	mit: MITEnroll
+	mit: MITEnroll,
+	cu: CornellEnroll
```
Comment (thatoddmailbox) [inline app/schools/EnrollModal.jsx:28]: Update ID to match the api-server

File: app/schools/SchoolSettingsModal.jsx [hunk 2]
```diff
@@ -5,13 +5,15 @@ import errors from "errors.js";
...
+import CornellSettings from "schools/cornell/CornellSettings.jsx";
 
 import LoadingIndicator from "ui/LoadingIndicator.jsx";
 import Modal from "ui/Modal.jsx";
 
 var settingsComponents = {
 	dalton: DaltonSettings,
-	mit: MITSettings
+	mit: MITSettings,
+	cu: CornellSettings
```
Comment (thatoddmailbox) [inline app/schools/SchoolSettingsModal.jsx:16]: Update ID to match the api-server

File: app/schools/cornell/CornellEnroll.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+				error: "You must enter your NetID Password."
+			});
+			return;
+		}
+
+		this.setState({
+			loading: true
+		}, () => {
+			api.post("schools/enroll", {
+				school: "cu",
```
Comment (thatoddmailbox) [inline app/schools/cornell/CornellEnroll.jsx:46]: Update ID to match the api-server

**Comment by willbarkoff** [comment]
🎊

**Pull request merged**